ydb/docs/en/core/postgresql/statements/create_table.md

```diff
@@ -7,7 +7,7 @@ The `CREATE TABLE` statement is used to create an empty table in the current dat
 {% include [syntax.md](../../../_includes/postgresql/statements/create_table/syntax.md) %}
 
 When creating a table, you can specify:
-1. **Table Type**: `TEMPORARY` (not fully supported yet and used only in tests) – a temporary table that is automatically deleted at the end of the session or at the end of the current transaction. If this parameter is not set (left empty), a permanent table is created.
+1. **Table Type**: `TEMPORARY` / `TEMP` – a temporary table that is automatically deleted at the regular end of the session, otherwise it will no longer be available and will be automatically deleted after the time has elapsed. If this parameter is not set (left empty), a permanent table is created.
```

Comment:
It's not clear what a "regular" session end means: it won't be dropped if there is an exception? What time has elapsed?